test/babelish/test_csv2android.rb

```diff
@@ -35,4 +35,18 @@ def test_converting_csv_to_dotstrings_one_output_option
     # clean up
     system("rm -rf ./" + single_file)
   end
+
+  def test_converting_with_comments
+    csv_file = "test/data/test_data_with_comments.csv"
+    french_file = "values-fr/strings.xml"
+    expected_output = File.read("test/data/test_data_fr_with_comments.xml")
+    converter = Babelish::CSV2Android.new(csv_file,
+    {'French' => "fr"},
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.